src/tools/rbd/action/Nbd.cc

```diff
@@ -187,19 +223,26 @@ int execute_unmap(const po::variables_map &vm)
   return call_nbd_cmd(vm, args);
 }
 
+int execute_unmap_depricated(const po::variables_map &vm)
+{
+  std::cerr << "rbd: 'nbd unmap' command is deprecated, "
+            << "use 'unmap -t nbd' instead" << std::endl;
+  return execute_unmap(vm);
+}
+
 Shell::SwitchArguments switched_arguments({"read-only"});
 
 Shell::Action action_show(
   {"nbd", "list"}, {"nbd", "ls"}, "List the nbd devices already used.", "",
-  &get_show_arguments, &execute_show);
+  &get_show_arguments, &execute_show_depricated, false);
 
 Shell::Action action_map(
   {"nbd", "map"}, {}, "Map image to a nbd device.", "",
-  &get_map_arguments, &execute_map);
+  &get_map_arguments, &execute_map_depricated, false);
 
 Shell::Action action_unmap(
   {"nbd", "unmap"}, {}, "Unmap a nbd device.", "",
-  &get_unmap_arguments, &execute_unmap);
+  &get_unmap_arguments, &execute_unmap_depricated, false);
```

Comment:
nit: deprecated

Reply:
Updated, thanks.

Also, the man page is updated.